### PR TITLE
Share Tafel Generator (frontend)

### PR DESCRIPTION
**Pull request by ovbm**
- [x] move Logo in Brand

### CODE REVIEW

**Review by ovbm — COMMENTED** [review]
Looks good IMO (can't approve because I started pr)

File: src/components/ShareImage/ShareImagePreview.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          (shareImage && shareImageJustify[textPosition]) || 'center',
+        borderWidth: embedPreview ? 2 : 0
+      }}
+    >
+      {format?.image && <img {...styles.formatImage} src={format?.image} />}
+      {format?.title && (
+        <div
+          {...styles.formatTitle}
+          style={{
+            color: coloredBackground ? '#FFF' : format?.color,
```
Comment (annatraussnig) [inline src/components/ShareImage/ShareImagePreview.js:116]: Color can also come from format kind -> needs to be added
`const formatColor = colorMeta && (colorMeta.color || colors[colorMeta.kind])`

**Review by tpreusse — COMMENTED** [review]
I really like the editing experience in Publikator. Here a few code comments. I think we should also do a naming iteration: see [backend review](https://github.com/orbiting/backends/pull/577#pullrequestreview-634020586).

File: src/components/ShareImage/docs.md [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+```
+
+# Share Image Generator
+
+A `<ShareImageGenerator />` takes a format type and returns a form to create a share image for that specific format type style.
+
+Supported props:
+
+- `socialKey`: required. Either `twitter` or `facebook`
+- `data` and `onInputChange`: an immutable `Map` and a setter. Both based on meta data model from Publikator
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply src/components/ShareImage/docs.md:79]: I see. Pfffff... Time to move some code around then. 😄

File: src/components/ShareImage/docs.md [hunk 3]
```diff
@@ -0,0 +1,236 @@
...
+        title: 'Briefings' 
+      }
+    },
+    color: '#00809C'
+  }}
+/>
+```
+
+## Dialog
+Setting type to 'Dialog' enables to choose between Sans-Serif and Serif fonts.
```
[the remark this answers is not in the document]
Comment (annatraussnig) [reply src/components/ShareImage/docs.md:195]: I leave that question to @ovbm

[the remark this answers is not in the document]
Comment (ovbm) [reply src/components/ShareImage/docs.md:195]: There seems to be a difference between the sketch file and the actual share images they are using... In the sketch file they use either sans-serif or serif. If in practice they only ever use meta, we can remove this.

[the remark this answers is not in the document]
Comment (tpreusse) [reply src/components/ShareImage/docs.md:195]: Let's remove it for now and re-evaluate if it comes up in practice. Same for meta, where we already removed it but the docs need an update:
Screenshot 2021-04-14 at 10 44 02
(Text talks about font switch but it's already removed)

File: src/components/ShareImage/docs.md [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+  inverted: false,
+  text: '',
+  format: { 
+    title: 'Aus der Arena',
+    section: {
+      meta: {
+        title: 'Format' 
+      }
+    },
+    kind: 'scribble'
```
Comment (ovbm) [inline src/components/ShareImage/docs.md:201]:
```suggestion
    kind: 'scribble',
    color: '#D74132'
```

Comment (ovbm) [reply src/components/ShareImage/docs.md:201]: needs a color so that background is correct afaik

Comment (annatraussnig) [reply src/components/ShareImage/docs.md:201]: does it? we use 
`const formatColor = format?.color || colors[format?.kind]`
to determine the color.

Comment (ovbm) [reply src/components/ShareImage/docs.md:201]: ok, so that doesn't work in this case. It's a black background without a specific color

Comment (annatraussnig) [reply src/components/ShareImage/docs.md:201]: aaaah for the background color. Got it

Comment (ovbm) [reply src/components/ShareImage/docs.md:201]: what if we do 
`  const formatBackgroundColor = format?.color || colors[format?.kind]
`
and
`        backgroundColor: inverted ? formatBackgroundColor : '#FFF',
`

Comment (ovbm) [reply src/components/ShareImage/docs.md:201]: ah forget that

Comment (ovbm) [reply src/components/ShareImage/docs.md:201]: formatColor is the way to go, thx!

Comment (annatraussnig) [reply src/components/ShareImage/docs.md:201]: pushed a fix

**Comment by ovbm** [comment]
LGTM

**Pull request merged**